heroku/cli
# chore(cli): add code coverage reports

## Why the change?
[GUS Card](https://gus.lightning.force.com/lightning/r/ADM_Work__c/a07EE00001NC3MDYA1/view)

As per our review of unit tests and coverage for the cli package, this PR does the following:
- Adds `nyc` package to add code coverage reports ✅
- Ensures that our code coverage reports over 85% test coverage of statements ✅

## Additional Notes
- Apparently sinon can't intercept/stub dependencies exported dependencies which has affected `file.ts` test coverage. Disregard in test coverage report.
- `user-config` mostly comprises of private members and methods which has affected test coverage. Disregard in test coverage report.

## How to verify?
1. Pull down branch
2. Run tests
3. Confirm test coverage report is visible and that it reports over 85% test coverage of statements for all files except for `file.ts` and `user-config.ts`
